META-INF/com/google/android/update-binary

```diff
@@ -270,14 +390,37 @@ patch_libbluetooth() {
 
 ui_print " "
 print_full_bar
-print_justified "Multi-disabler v3.1 for Samsung devices"
+print_justified "Multi-disabler for Samsung devices"
 print_justified "running Android 9 or later."
-print_justified "by Ian Macdonald"
+print_justified "by Ian Macdonald, afaneh92"
+print_justified "corsicanu & Simon1511"
```

Comment:
It would be more coherent to have the names of the contributors if they agree, to be consistent with the name of initial developer

Reply:
Who are you?